pkgs/tools/misc/tere/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, fetchFromGitHub, rustPlatform }:
+
+let
+  commonMeta = {
+    name = "tere";
+    version = "1.1.0";
+  };
+  tereSrc = fetchFromGitHub {
+    owner = "mgunyho";
+    repo = commonMeta.name;
+    rev = "v${commonMeta.version}";
+    sha256 = "BD7onBkFyo/JAw/neqL9N9nBYSxoMrmaG6egeznV9Xs=";
+  };
+in
```

Comment:
These are basically useless since you only use them once in the `in` block anyways ?

It's definitely cleaner to just remove these and inline